### DISCUSSION TITLE
DIN Letter Template?

### DISCUSSION THREAD

**Discussion by ondohotola** [typst/typst#324] [Q&A] [answered]
Hi,

does anyone have a DIN Letter Template (of sorts), like the KOMA-Letter in LaTeX?

**Comment by reknih** [comment, marked as answer]
Hey,

**Edited**
here is a DIN letter template similar based on our internal letters: 
https://gist.github.com/reknih/a568bdda88507cbe021e4d11f92b9132

**Original message**
we use a DIN letter template internally for our company mail. I could upload it to a Gist and share it here after I removed our company info.

**Reply by Corbie-42** [reply]
Thank you, this is a good start, although it does not fit the DIN 5008 yet.

**Comment by ondohotola** [comment]
please do

el


--
Dr. Eberhard W. Lisse \ / Obstetrician & Gynaecologist
***@***.*** / * | Telephone: +264 81 124 6733 (cell)
PO Box 8421 Bachbrecht \ / If this email is signed with GPG/PGP
10007, Namibia ;____/ Sect 20 of Act No. 4 of 2019 may apply

**Reply by reknih** [reply]
I updated my previous reply with the template!

**Comment by ondohotola** [comment]
Martin,

thanks, I have downloaded.

I can certainly adapt that for my own use and it'll give me a reason to
learn typest's DSL.

Gruss aus dem Süden,

el
